Automattic/wp-calypso
# Hosting dashboard: clean up performance tab

Part of:

- https://github.com/Automattic/dotcom-forge/issues/7924

## Proposed Changes

This PR refactors the code for the hosting dashboard's Performance tab, by moving them under `client/hosting/performance/` directory.

## Why are these changes being made?

Consistency and to avoid future confusion.

## Testing Instructions

1. Go to `<Calypso Live URL>/site-performance/:siteSlug` (of an Atomic site)
2. Smoke test and verify that the feature still works.

## Pre-merge Checklist
